resolver-functions/CreateBulkDownload/CreateBulkDownload.ts

```diff
@@ -43,106 +43,12 @@ export const CreateBulkDownloadResolver = async (root, args, context, info) => {
       },
       workflow_run_ids: workflowRunIdsNumbers ?? workflowRunIds,
     };
-    const railsResponse = await postWithCSRF({
+    const res = await postWithCSRF({
       url: `/bulk_downloads`,
       body,
       args,
       context,
     });
-    if (railsResponse.error != null) {
-      throw new Error(railsResponse.error);
-    }
-    return {
-      id: railsResponse.id.toString(),
-    };
-  }
-
-  /* --------------------- Next Gen --------------------- */
-  // get the default bulk download workflow version id from the workflow service
-  if (!workflowRunIdsStrings || workflowRunIdsStrings.length === 0) {
-    throw new Error(
-      "No Next Gen Workflow Ids provided for bulk download creation",
-    );
+    return res;
```

Comment:
I think you'll have to change the json schema to make `id` not `required`.

could also just keep the `error` check and just `return;`?

Reply:
Yeah - that sounds good. Thanks!